sql/maintenance_utils.sql

```diff
@@ -39,11 +39,24 @@ CREATE OR REPLACE FUNCTION @extschema@.compress_chunk(
     hypercore_use_access_method BOOL = NULL
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C VOLATILE;
 
+-- Alias for compress_chunk above.
+CREATE OR REPLACE PROCEDURE @extschema@.convert_to_columnstore(
+    uncompressed_chunk REGCLASS,
+    if_not_columnarstore BOOLEAN = true,
+    recompress BOOLEAN = false,
```

Comment:
Not sure what the recompress parameter is used for.  Can we get rid of it?


FWIW, with the new naming it is not clear that one can call this function again to "recompress".

Reply:
Or even `update_settings_and_recompress`...

Reply:
Honestly, w.r.t. to the `recompress` parameter, I think decompressing and compressing manually or in a script is the way to go. It also makes it more clear what is happening and the consequences. It is exactly what is happening internally anyway.

If removing it turns out to be a mistake, we can always add it back.

Reply:
What about `rebuild` instead of `recompress`?